docs/herbarium/working-with-herbarium.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
Since Herbarium is quite straightforward, I think it could have just one page instead of two, as it is on this PR